lib/connection/spawnInterruptibleJulia.ps1

```diff
@@ -1,11 +1,11 @@
 param(
-	[Int32] $port, 
-	[string] $jlpath, 
-	[string] $jloptions
+	[Int32] $port,
+	[string] $jlpath,
+	[string] $boot
 )
 
 # start Julia
-$proc = Start-Process $jlpath "$jloptions -e `"import Atom; @sync Atom.connect($port)`"" -NoNewWindow -PassThru
+$proc = Start-Process $jlpath @($boot, $port) -NoNewWindow -PassThru
```

Comment:
Is this valid?
Also, if all goes well, this file won't survive for long and be superseded be something actually working... :)